code/game/objects/items/plushes.dm

```diff
@@ -118,9 +119,22 @@
 			if(!stuffed)
 				to_chat(user, "<span class='warning'>You already murdered it!</span>")
 				return
-			user.visible_message("<span class='notice'>[user] tears out the stuffing from [src]!</span>", "<span class='notice'>You rip a bunch of the stuffing from [src]. Murderer.</span>")
-			I.play_tool_sound(src)
-			stuffed = FALSE
+			if(!divine)
+				user.visible_message("<span class='notice'>[user] tears out the stuffing from [src]!</span>", "<span class='notice'>You rip a bunch of the stuffing from [src]. Murderer.</span>")
+				I.play_tool_sound(src)
+				stuffed = FALSE
+			else
+				to_chat(user, "<span class='notice'>What a fool you are. [src] is a god, how can you kill a god? What a grand and intoxicating innocence.</span>")
+				if(iscarbon(user) && user.drunkenness < 50)
```

Comment:
it seems like the automatic checks don't like me referencing the drunkenness variable with a living mob, even though I checked for carbon-ness beforehand

is there a way around this? like, a way to temporarily typecast user as a carbon or something?